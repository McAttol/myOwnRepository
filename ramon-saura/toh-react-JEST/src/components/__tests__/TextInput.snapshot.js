import renderer from "react-test-renderer";
import React from "react";
import TextInput from "../TextInput";
import { BrowserRouter as Router } from 'react-router-dom';


describe ('Text Input snapshot', () => {
    const textInput = renderer.create(<Router><TextInput/></Router>)

    it('shoul match', () => {
        expect(textInput.toJSON()).toMatchSnapshot();
    })
});