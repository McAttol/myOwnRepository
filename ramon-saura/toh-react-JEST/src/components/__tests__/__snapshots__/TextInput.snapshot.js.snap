// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text Input snapshot shoul match 1`] = `
<label>
  Name: 
  <input
    type="text"
  />
</label>
`;
